packaging/test_plugin_unified_storage_tiering.py

```diff
@@ -42,10 +42,17 @@ def storage_tiering_configured_custom(arg=None, sleep_time=1):
 
         irods_config.commit(irods_config.server_config, irods_config.server_config_path)
         try:
+            # Reload configuration after edits are made so that they take effect in the server.
+            IrodsController().reload_configuration()
             yield
+
         finally:
             pass
 
+    # Reload configuration after exiting the context so that the original settings take effect.
+    IrodsController().reload_configuration()
```

Comment:
I'm thinking we might be able to put the configuration reload in the `finally` clause, but I got spooked by some failing tests and didn't feel like running all the tests again after they were passing in this state. :) Can update if desired.

Reply:
I don't have a strong preference. They appear to result in identical behavior so I think you get to pick your favorite solution.

Reply:
I looked at it again and this is actually the correct implementation. It has to be outside of the `file_backed_up` context manager so that the server_config is actually back to its original state before we reload the configuration. If the call was made in the `finally` block, server_config would still be in its modified state. Resolving.